src/Mailer/Ownership/SendOwnershipMailCommandHandler.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Ownership;
+
+use Broadway\CommandHandling\CommandHandler;
+use CultuurNet\UDB3\Iri\IriGeneratorInterface;
+use CultuurNet\UDB3\Mailer\Command\SendOwnershipRequestedMail;
+use CultuurNet\UDB3\Mailer\Mailer;
+use CultuurNet\UDB3\Mailer\MailsSentRepository;
+use CultuurNet\UDB3\Model\ValueObject\Identity\Uuid;
+use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddress;
+use CultuurNet\UDB3\Ownership\Repositories\OwnershipItemNotFound;
+use CultuurNet\UDB3\Ownership\Repositories\Search\OwnershipSearchRepository;
+use CultuurNet\UDB3\ReadModel\DocumentDoesNotExist;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\User\UserIdentityResolver;
+use DateTimeImmutable;
+use Psr\Log\LoggerInterface;
+use Twig\Environment as TwigEnvironment;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Error\SyntaxError;
+
+class SendOwnershipMailCommandHandler implements CommandHandler
+{
+    private const SUBJECT_OWNERSHIP_REQUESTED = 'Beheers aanvraag voor organisatie {{ organisationName }}';
+    private const TEMPLATE_OWNERSHIP_REQUESTED = 'ownershipRequested';
+
+    private Mailer $mailer;
+    private MailsSentRepository $mailsSentRepository;
+    private LoggerInterface $logger;
+    private DocumentRepository $organizerRepository;
+    private UserIdentityResolver $identityResolver;
+    private IriGeneratorInterface $organizerIriGenerator;
+    private TwigEnvironment $twig;
+    private OwnershipSearchRepository $ownershipSearchRepository;
+
+    public function __construct(
```

Comment:
This class contains a lot of injected interfaces. Maybe it makes sense to create a class that extracts the required parameters for the twig template. That allows for much more readable code, easier-to-write tests and easier-to-maintain code.